pkg/operation/botanist/controlplane.go

```diff
@@ -129,7 +140,16 @@ func (b *Botanist) DeployVerticalPodAutoscaler(ctx context.Context) error {
 	}
 	values["global"] = map[string]interface{}{"images": values["images"]}
 
-	return b.K8sSeedClient.ChartApplier().Apply(ctx, filepath.Join(charts.Path, "seed-bootstrap", "charts", "vpa", "charts", "runtime"), b.Shoot.SeedNamespace, "vpa", kubernetes.Values(values))
+	if err := b.K8sSeedClient.ChartApplier().Apply(ctx, filepath.Join(charts.Path, "seed-bootstrap", "charts", "vpa", "charts", "runtime"), b.Shoot.SeedNamespace, "vpa", kubernetes.Values(values)); err != nil {
+		return err
+	}
+
+	// TODO(rfranzke): Remove in a future release.
+	return kutil.DeleteObjects(ctx, b.K8sSeedClient.Client(),
```

Comment:
You may also want to clean-up the renamed RBAC resources, right?

Reply:
This will be done by GRM since I only renamed the resources in the shoot. For the RBAC resources in the seed, I suggested to do this as part of the Helm -> component refactoring (#2754).

Reply:
Okay got it, thanks for clarifying.